package.json

```diff
@@ -43,6 +43,7 @@
   },
   "homepage": "https://github.com/recharts/react-smooth#readme",
   "peerDependencies": {
+    "prop-types": "^15.5.8",
```

Comment:
It's recommended that this should live inside dependencies https://github.com/reactjs/prop-types#how-to-depend-on-this-package